app/views/incomeSources/add/IncomeSourceReportingMethodNotSaved.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.layouts.unifiedLayout
+@import views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF
+@import enums.IncomeSourceJourney.{ForeignProperty, IncomeSourceType, SelfEmployment, UkProperty}
+
+@this(
+mainTemplate: unifiedLayout,
+h1: h1,
+p: p,
+formWithCSRF: FormWithCSRF,
+continueButton: ContinueButton,
+)
+
+@(isAgent: Boolean = false, incomeSourceType: IncomeSourceType, continueAction: Call)(implicit user: auth.MtdItUser[_], messages: Messages)
+
+@businessType = @{
+  incomeSourceType match {
+    case  _:SelfEmployment.type => messages("incomeSources.add.error.reportingMethodNotSaved.se")
+    case _:UkProperty.type => messages("incomeSources.add.error.reportingMethodNotSaved.uk")
+    case _:ForeignProperty.type => messages("incomeSources.add.error.reportingMethodNotSaved.fp")
+  }
+}
+
+@mainTemplate(pageTitle = messages("incomeSources.add.error.standardError"), isAgent = isAgent, backUrl = None, btaNavPartial = user.btaNavPartial) {
+
+@h1(msg = "incomeSources.add.error.standardError", classes="govuk-heading-l govuk-!-margin-bottom-5")
+
+@p(id = Some("paragraph-1")){@messages("incomeSources.add.error.reportingMethodNotSaved.p1", businessType)}
+
+@p(id = Some("paragraph-2")){@messages("incomeSources.add.error.reportingMethodNotSaved.p2")}
+
+@formWithCSRF(action = continueAction , Symbol("id") -> "error-reporting-method-not-saved-form") {
+    @continueButton(Some(messages("base.continue")))
+}
```

Comment:
should be indented as within mainTemplate block, also would prefer for messages within { to have their own line for readability